src/ws_stream_wasm.rs

```diff
@@ -29,6 +33,7 @@ impl crate::next::Connection for Connection {
     async fn receive(&mut self) -> Option<crate::next::Message> {
         use crate::next::Message;
         loop {
+            #[allow(clippy::match_same_arms)] // FIXME Remove when MSRV >= 1.78.0
```

Comment:
Why "Remove when MSRV >= 1.78.0"?

Reply:
Rust 1.76 (clippy) does not detect that the match arms have different meanings. With newer Rust versions this is fixed.

Reply:
Can you just remove this please?  It's a pedantic lint and I do not want pedantic lints